inference-engine/include/ie_parameter.hpp

```diff
@@ -20,7 +20,7 @@
 #include <vector>
 
 #include "ie_blob.h"
-#include <ngraph/ngraph.hpp>
+#include <ngraph/variant.hpp>
```

Comment:
is it needed? We don't have `ngraph::Variant` integration here

Reply:
Good catch. Looks like variant support was removed